### PR TITLE
MINOR: Move `Os` class to utils package and rename it to OperatingSystem

### PR DESCRIPTION
**Pull request by ijuma**
The `common` package is public and this class is
internal.

### CODE REVIEW

**Comment by ijuma** [comment]
I only noticed this after merging https://github.com/apache/kafka/commit/42284960dab4724445a4a3a7831b3b48f0ddcb48. @hachikuji, can you please review this? cc @cmccabe

File: clients/src/main/java/org/apache/kafka/common/utils/Os.java [hunk 1]
```diff
@@ -14,11 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.common;
+package org.apache.kafka.common.utils;
 
 import java.util.Locale;
 
 public final class Os {
```
Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/common/utils/Os.java]: Could we rename this to `OperatingSystem`?

Comment (ijuma) [reply clients/src/main/java/org/apache/kafka/common/utils/Os.java]: Done.

**Review by hachikuji — APPROVED** [review]
LGTM. Thanks for the patch!

**Pull request closed without merging**